app.config.yaml

```
@@ -132,7 +132,8 @@ application:
         events_of_interest:
           - provider_metadata: dx_commerce_events
             event_codes:
-              - com.adobe.commerce.observer.checkout_oope.sales_order_creditmemo_save_after
+              # com.adobe.commerce.<your-application-name>.<commerce-event-code> eg: your-application-name = app_one
+              - com.adobe.commerce.app_one.observer.sales_order_creditmemo_save_after
```

Just my preference, I slightly prefer using 'Checkout Starter Kit' as the default name for this project, but your call:
```suggestion
              - com.adobe.commerce.checkout_starter_kit.observer.sales_order_creditmemo_save_after
```

---

Yeah I forgot to revert back to `checkout_starter_kit`. During internal demo I updated that.